data/mods/Xedra_Evolved/items/comestibles/goblin_fruits.json

```diff
@@ -0,0 +1,313 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "goblin_fruit_abstract",
+    "name": { "str": "Abstract Goblin Fruit" },
+    "description": "Somehow you got a hold of the platonic goblin fruit, the one fruit to rule them all.  It's a bug if you have this, but the lords of the Bright Lands will surely pay a high price for it.",
```

Comment:
I chuckled at this line.  Also just an FYI, XE has added some additional fruit bearing trees at some spillage locations.  Figs, pomegranates and Hawthorns.  In case you were looking for other goblin fruit types.